src/aktualizr_info/main.cc

```diff
@@ -24,6 +44,7 @@ int main(int argc, char **argv) {
     ("ecu-keys",  "Outputs UPTANE keys")
     ("images-root",  "Outputs root.json from images repo")
     ("images-target",  "Outputs targets.json from images repo")
+    (delegation_opt_name.c_str(),  "Outputs metadata of image repo targets' delegations")
```

Comment:
I see the point but having this option using a different format here is distracting. And we don't go and change options names everyday, so the benefit of this indirection compared to the readability cost is disputable.

Also, a `std::string` looks overkill here.

Reply:
makes sense. will update it to make it consistent with the other options.